docs/online/src/scenarios/postmessage.rst

```diff
@@ -397,6 +465,35 @@ every outgoing PostMessage:
             }
         }
 
+..  data:: App_RefreshSessionInfo
+
+    The :term:`App_RefreshSessionInfo` message is poseted when the app detects the need of new tokens. The host responds to this by
+    sending a response message :term:`App_RefreshSessionInfo` with a :term:`Host_RefreshSessionInfo` response. 
+
```

Comment:
Please add some blurb that this was designed for requesting new WOPI access tokens. Since some apps will need more than that, we made it support more than that. As of today, the only tokens for which we have implemented support are: AccessToken (Word, Excel, PowerPoint) and FileGetUrl (PowerPoint).

Reply:
@tylerbutler is this the kind of detail level we want?